mappings/net/minecraft/entity/data/DataTracker.mapping

```diff
@@ -1 +1,30 @@
 CLASS net/minecraft/class_216 net/minecraft/entity/data/DataTracker
+	FIELD field_1740 DATA_TYPE_MAP Ljava/util/HashMap;
+	FIELD field_1741 trackedData Ljava/util/Map;
```

Comment:
`entries` according to yarn - https://github.com/FabricMC/yarn/blob/4674b66184231baebe54ca8bf95568fd677f25f8/mappings/net/minecraft/entity/data/DataTracker.mapping#L3